docs/src/cli/generate.md

```diff
@@ -38,6 +38,10 @@ What generated files to emit. Possible values:
 - `parser` (default): Generate `parser.c` and related files.
 - `lib`: Compile to a library (equivalent of the deprecated `--build` option)
 
+### `-b/--build`
```

Comment:
I remember there being some doubt about this flag. I think this was motivated by split/multiple parser projects?

Would it not be better to have the CLI check and rebuild all parsers? It's a more complex but robust (and useful) solution, so it might make sense to not prematurely add this API, only to deprecate it later.

Reply:
But you are right: This deprecation would be an independent decision (and should be an independent discussion), so not blocking this PR (or the release).